tests/ProjectFixerConfigurationTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests;
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @author SpacePossum
+ *
+ * @internal
+ *
+ * @covers .php_cs.dist
+ */
+final class ProjectFixerConfigurationTest extends TestCase
+{
+    public function testCreate()
+    {
+        $setLocation = __DIR__.'/../.php_cs.dist';
+
+        /** @var \PhpCsFixer\Config $config */
+        $config = require $setLocation;
+
+        $this->assertInstanceOf('PhpCsFixer\Config', $config);
+        $this->assertEmpty($config->getCustomFixers());
+        $this->assertNotEmpty($config->getRules());
+    }
+}
```

Comment:
it's not testing our package as application/library, it's checking it's own config file that is not used when executing code as app/lib on user side.
imo, AutoReview here